src/controllers/connectionManager.ts

```diff
@@ -265,10 +265,11 @@ export default class ConnectionManager {
 	 * @param includePassword (optional) if password should be included in connection string.
 	 * @returns connection string for the connection
 	 */
-	public async getConnectionString(connectionUri: string, includePassword: boolean = false): Promise<string> {
+	public async getConnectionString(connectionUri: string, includePassword: boolean = false, includeApplicationName: boolean = true): Promise<string> {
```

Comment:
What's the plan for actually updating the version of STS?

Reply:
I would like to try and get my other connectionString improvements in as well prior to updating the version of STS.

Reply:
Are you implying you're going to merge this in before updating STS? Generally we shouldn't do that - once there's a build of STS with your changes you should include it in this PR and then merge it in.

Reply:
@Charles-Gagnon I'll follow that approach so that we can get these changes into the current insiders.

Reply:
What approach? If you merge it in now the STS changes aren't going to be there so this won't actually do anything until STS is updated to a version that has your changes.

Reply:
The approach you stated in your previous comment. Merge STS, update version of STS, and then use the latest STS version. I don't see any docs on how to go about that. Is there a standard process for it?

Reply:
https://github.com/microsoft/azuredatastudio/pull/18212